Deployment/templates/continuous-deployment.yml

```
@@ -0,0 +1,43 @@
+parameters:
+  - name: ContinueEvenIfResourcesAreGettingDestroyed
+    type: boolean
+    default: false
+  - name: AzureSubscription
+    type: string
+
+steps:
+  - task: FileTransform@1
+    displayName: "File Transform: Ens Config" #Replace exchange set instance value from pipeline 
+    inputs:
+      folderPath: '$(Pipeline.Workspace)/terraformartifact/src'
+      fileType: 'json'
+      targetFiles: 'appsettings.json'
+
+  - task: PowerShell@2
+    displayName: "terraform devdeploy"
```

Can we use the right environment name here please depending of which stage is using this template, maybe?:
```suggestion
    displayName: "terraform $(Environment)deploy"
```

---

Thanks